tests/test_fixity.py

```diff
@@ -462,6 +462,5 @@ def test_main_handles_exceptions_if_scanall_fails(_get, monkeypatch, capsys):
 
     captured = capsys.readouterr()
     assert (
-        captured.out.strip()
-        == f"Internal error encountered while scanning AIP {aip_id} (StorageServiceError)"
-    )
+        f"Internal error encountered while scanning AIP {aip_id} (StorageServiceError)\n"
+    ) in captured.out
```

Comment:
Why is this change necessary?

Reply:
just to keep uniformity same as above assertions for `test_scanall_handles_exceptions`

Reply:
I see. Good thinking, but maybe we could do the opposite: change the `in` comparison in `test_scanall_handles_exceptions` to be an equality comparison with `==`.

We want exact matches as much as possible, don't you think?